README.rst

```diff
@@ -52,15 +52,65 @@ Quick Example
     driver.close()
 
 
+Connection Settings Breaking Change
+===================================
+
++ The driver’s default configuration for encrypted is now false (meaning that driver will only attempt plain text connections by default).
+
++ Connections to encrypted services (such as Neo4j Aura) should now explicitly be set to encrypted.
+
++ When encryption is explicitly enabled, the default trust mode is to trust the CAs that are trusted by operating system.
+
++ This means that encrypted connections to servers holding self-signed certificates will now fail on certificate verification by default.
+
++ Hostname verification is turned on by default when encryption is turned on.
+
++ The `neo4j://` scheme replaces `bolt+routing://` and can be used for both clustered and single-instance configurations with Neo4j 4.0.
+
+
+See, https://neo4j.com/docs/migration-guide/4.0/upgrade-driver/#upgrade-driver-breakingchanges
```

Comment:
Also add a link to the manual, where this is described in a lot more detail:
https://neo4j.com/docs/driver-manual/current/client-applications/#driver-connection-uris